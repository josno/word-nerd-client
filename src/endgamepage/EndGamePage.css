.home-page-title,
.end-message,
.game-instructions,
.input-instructions {
	margin-top: 10vh;
}

.game-controls {
	display: flex;
	justify-content: space-around;
}
