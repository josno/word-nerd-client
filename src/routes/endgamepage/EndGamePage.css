.end-message {
	margin-top: 30px;
}

.home-button {
	display: block;
	font-size: 0.73em;
	margin-left: auto;
	margin-right: auto;
	margin-top: 10px;
	transition: all 0.5s ease;
	width: 50vw;
	border-radius: 10px;
	border: none;
	background-color: #ffbc42;
	font-weight: bold;
}

.home-button:hover {
	cursor: pointer;
}

.home-button:active {
	background-color: #ba8931;
}
