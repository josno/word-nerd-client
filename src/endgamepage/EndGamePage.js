import React from 'react';
import './EndGamePage.css';

export default function EndGamePage() {
	return (
		<div className="end-game-page">
			<h1 class="end-message">Game Over</h1>
			<h2>Thanks for playing!</h2>
			<div class="game-controls">
				<a href="three-input-page.html">
					<button class="play-again">Play Again</button>
				</a>
				<a href="two-game-home-page.html">
					<button class="home-button">Go Home</button>
				</a>
			</div>
		</div>
	);
}
