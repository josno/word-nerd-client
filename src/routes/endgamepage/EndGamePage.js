import React from 'react';
import { Link } from 'react-router-dom';
import './EndGamePage.css';

export default function EndGamePage() {
	return (
		<main className="end-game-page">
			<header>
				<h1 className="end-message">Game Over</h1>
			</header>
			<h2>Thanks for playing!</h2>
			<section className="end-game-controls">
				<Link to="/game-home-page">
					<button aria-label="go-back" className="home-button">
						Go Back To Games Home
					</button>
				</Link>
			</section>
		</main>
	);
}
