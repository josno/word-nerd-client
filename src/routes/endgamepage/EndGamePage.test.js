import React from 'react';
// import ReactDOM from 'react-dom';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import EndGamePage from './EndGamePage';

describe('<EndGamePage/>', () => {
	it('renders as expected', () => {
		const tree = renderer
			.create(
				<BrowserRouter>
					<EndGamePage />
				</BrowserRouter>
			)
			.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
