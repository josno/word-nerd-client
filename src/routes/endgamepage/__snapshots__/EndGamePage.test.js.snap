// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EndGamePage/> renders as expected 1`] = `
<main
  className="end-game-page"
>
  <header>
    <h1
      className="end-message"
    >
      Game Over
    </h1>
  </header>
  <h2>
    Thanks for playing!
  </h2>
  <section
    className="end-game-controls"
  >
    <a
      href="/game-home-page"
      onClick={[Function]}
    >
      <button
        aria-label="go-back"
        className="home-button"
      >
        Go Back To Games Home
      </button>
    </a>
  </section>
</main>
`;
